index.bs

```diff
@@ -2468,6 +2520,61 @@ devices=] that are rarely used or do not include a [=platform authenticator=]; o
+    and returned to the [=authenticator=] via the {{PublicKeyCredentialRequestOptions/allowCredentials}} option of
+    {{CredentialsContainer/get()}}, which allows the [=authenticator=] to decrypt and use the [=credential private key=].
+
+    This enables the [=authenticator=] to have unlimited storage capacity for [=credential private keys=], since the encrypted
+    [=credential private keys=] are stored by the [=[RP]=] instead of by the [=authenticator=] - but it means that a
+    [=credential=] stored in this way must be retrieved from the [=[RP]=] before the [=authenticator=] can use it.
+
+Which of these storage strategies an [=authenticator=] supports defines the [=authenticator=]'s <dfn>storage modality</dfn> as
+follows:
+
+- An [=authenticator=] has the <dfn>local storage modality</dfn> if it supports storing [=credential private keys=] in persistent
+    storage embedded in the [=authenticator=], or if it supports both storage strategies. An [=authenticator=] with [=local
+    storage modality=] is also called <dfn>local storage capable</dfn>.
+
+- An [=authenticator=] has the <dfn>remote storage modality</dfn> if it does not have the [=local storage modality=], i.e., it
+    only supports storing [=credential private keys=] as a ciphertext in the [=credential ID=].
+
+Note that an [=authenticator=] with [=local storage modality=] MAY support both storage strategies. In this case, the
+[=authenticator=] MAY at its discretion use different storage strategies for different [=public key credential|credentials=],
+though subject to the {{AuthenticatorSelectionCriteria/requireResidentKey}} option of {{CredentialsContainer/create()}}.
+
+
+### <dfn>Authentication factor capability</dfn> ### {#sctn-authentication-factor-capability}
+
+There are three broad classes of [=authentication factors=] that can be used to prove an identity during an [=authentication
+ceremony=]: [=something you have=], [=something you know=] and [=something you are=]. Examples include a physical key, a password,
+and a fingerprint, respectively.
+
+This specification is mainly concerned with the [=something you have=] class - an [=authenticator=] containing a [=credential
+private key=] is such an [=authentication factor=]. However, an [=authenticator=] that supports [=user verification=] can also act
+as one or two additional kinds of [=authentication factor=]. For example, if the [=authenticator=] can verify a PIN, the PIN is
+[=something you know=], and a [=biometric authenticator=] can verify [=something you are=].
+
+Since the [=authenticator=] itself is [=something you have=], this means that an [=authenticator=] that supports [=user
+verification=] can act as more than one kind of [=authentication factor=] - possibly all three kinds. Therefore, an
+[=authenticator=] that supports [=user verification=] is <dfn>multi-factor capable</dfn>. Conversely, an [=authenticator=] that is
+not [=multi-factor capable=] is <dfn>single-factor capable</dfn>.
```

Comment:
<dfn>single-factor capable</dfn>, i.e., <dfn>second-factor only</dfn>.

Reply:
sort of a nit, but I'd try to figure out how to combine the middle two parags -- their content overlaps currently.

Reply:
I combined the two paragraphs, but I'm not sure how to do the "second-factor only" thing in a way that both makes sense, is correct and not overly verbose. :) I mean, it's not unreasonable that some RPs might want to allow single-factor login with "second-factor" authenticators, so "second-factor only" is not quite accurate. So I'm thinking something like "i.e., usable only as a second factor for multi-factor authentication", but at that point it feels like it gets a little too dense and verbose, and it might be better to just leave it all out.

WDYT?